examples/typescript/aws-s3-static-website/main.ts

```diff
@@ -0,0 +1,107 @@
+import { Construct } from "constructs";
+import { App, TerraformStack, TerraformVariable, TerraformAsset, TerraformOutput } from "cdktf";
+import { AwsProvider, s3 } from "./.gen/providers/aws";
```

Comment:
It looks like these imports need to be updated with the namespace changes that we introduced with our 0.14 release. Look here for an example: https://github.com/hashicorp/terraform-cdk/blob/main/examples/typescript/aws-multiple-stacks/main.ts#L5-L6